### PR TITLE
Distinguish error from warning on debug page

### PR DESCRIPTION
**Pull request by arothuis**
[129997131](https://www.pivotaltracker.com/story/show/129997131)
OpenConext/OpenConext-engineblock#207

### CODE REVIEW

**Comment by arothuis** [comment]
I have not yet been able to test this visually. Also, the texts have not not changed yet.

**Comment by thijskh** [comment]
alt tag value should probably  be `&#x274c;`

File: app/Resources/views/modules/Authentication/View/Proxy/debugidpresponse.phtml [hunk 1]
```diff
@@ -44,7 +44,7 @@ $validationResult = EngineBlock_ApplicationSingleton::getInstance()
           <?= $this->t('attributes_validation_succeeded'); ?>
       <?php else: ?>
           <?php if (count($validationResult->getErrors()) > 0): ?>
-            <img src="/images/warning.svg" class="c-icon c-icon-warning" alt="&#9888;" />
+            <img src="/images/error.svg" class="c-icon c-icon-error" alt="&#9888;" />
```
Comment (rjkip) [inline app/Resources/views/modules/Authentication/View/Proxy/debugidpresponse.phtml]: The alt attribute contains the warning character here.

File: app/Resources/views/modules/Authentication/View/Proxy/debugidpresponse.phtml [hunk 2]
```diff
@@ -44,7 +44,7 @@ $validationResult = EngineBlock_ApplicationSingleton::getInstance()
           <?= $this->t('attributes_validation_succeeded'); ?>
       <?php else: ?>
           <?php if (count($validationResult->getErrors()) > 0): ?>
```
Comment (rjkip) [inline app/Resources/views/modules/Authentication/View/Proxy/debugidpresponse.phtml]: Interesting if-statement that was :sweat_smile: If true then warning, if false then warning.

File: theme/material/templates/modules/Authentication/View/Proxy/debugidpresponse.phtml [hunk 3]
```diff
@@ -43,7 +43,7 @@ $validationResult = EngineBlock_ApplicationSingleton::getInstance()
           <?= $this->t('attributes_validation_succeeded'); ?>
       <?php else: ?>
           <?php if (count($validationResult->getErrors()) > 0): ?>
-            <img src="/images/warning.svg" class="c-icon c-icon-warning" alt="&#9888;" />
+            <img src="/images/error.svg" class="c-icon c-icon-error" alt="&#9888;" />
```
Comment (rjkip) [inline theme/material/templates/modules/Authentication/View/Proxy/debugidpresponse.phtml]: Here as well.

Comment (arothuis) [inline theme/material/templates/modules/Authentication/View/Proxy/debugidpresponse.phtml]: Fixed it.

**Pull request merged**